api_examples/paddle/fluid/layers/reduce_op.py

```diff
@@ -0,0 +1,37 @@
+import paddle.fluid as fluid
```

Comment:
You should follow example template:)

Reply:
Add copyright, api description and comments